Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -253,6 +263,16 @@ void WiiPane::OnSaveConfig()
   Config::SetBase(Config::MAIN_ALLOW_SD_WRITES, m_allow_sd_writes_checkbox->isChecked());
   Settings::Instance().SetUSBKeyboardConnected(m_connect_keyboard_checkbox->isChecked());
 
+  std::string nus_url = m_nus_url_edit->text().toStdString();
+  if (nus_url.compare(0, 4, "http"))
```

Comment:
Since `StringUtil.h` is already included you might want to use `StringBeginsWith` if you decide to stay with `std::string`.